frontend/src/app/app-routing.module.ts

```diff
@@ -28,7 +32,7 @@ const routes: Routes = [
     children: [
       // experiments
       { path: '', redirectTo: 'all', pathMatch: 'full' },
-      { path: 'all', component: AllExperimentListComponent},
+      { path: 'all', component: PublicExperimentListComponent},
```

Comment:
Rename this path to `public`

Reply:
Don't forget to change the `all` routes to `public` in `src/app/components/experiments.component.html` as well